_sass/template/print-pdf.scss

```diff
@@ -380,72 +381,75 @@ $print-page-setup-lightning-source: false !default; // Removes crop marks and se
-@import "template/partials/print-bibliographies";
-@import "template/partials/print-dialogue";
-@import "template/partials/print-epigraphs";
-@import "template/partials/print-dedications";
-@import "template/partials/print-glossaries"; // Including .glossary
-@import "template/partials/print-highlighter";
-@import "template/partials/print-pullquotes"; // Supports .pullquote
-@import "template/partials/print-boxes";
-@import "template/partials/print-code"; // For computer code and preformatted text
-@import "template/partials/print-tables";
-@import "template/partials/print-figures";
-@import "template/partials/print-notes"; // Footnotes, endnotes, sidenotes
-@import "template/partials/print-previous-publications-page"; // Also by the author page in frontmatter
-@import "template/partials/print-title-pages"; // Half-title and title pages
-@import "template/partials/print-copyright-page";
-@import "template/partials/print-toc"; // Tables of contents
-@import "template/partials/print-video";
-@import "template/partials/print-cover"; // Front-cover image for PDF ebooks
-@import "template/partials/print-smallcaps";
-@import "template/partials/print-maths";
-@import "template/partials/print-letters";
-@import "template/partials/print-sources";
-@import "template/partials/print-openers";
-@import "template/partials/print-index";
-@import "template/partials/print-questions";
-@import "template/partials/print-select-questions";
-@import "template/partials/print-cross-refs";
-@import "template/partials/print-slides";
-@import "template/partials/print-reset-sequences"; // Resets p indents, margins after other elements. Must be last @import in list.
-@import "template/partials/print-page-headers-footers-content"; // Sets content for headers and footers
-@import "template/partials/print-page-headers-footers-style"; // Sets styling for headers and footers
+@import "template/partials/pdf-start-on-recto-or-verso"; // Starts chapters on a left- or right-hand-page
+@import "template/partials/pdf-page-start-depth"; // Sets a top margin on opening pages
+@import "template/partials/pdf-base-typography"; // Default typography for HTML elements
+@import "template/partials/pdf-columns";
+@import "template/partials/pdf-buttons";
+@import "template/partials/pdf-verse"; // Default typography for verse, poetry, lyrics
+@import "template/partials/pdf-bibliographies";
+@import "template/partials/pdf-dialogue";
+@import "template/partials/pdf-epigraphs";
+@import "template/partials/pdf-dedications";
+@import "template/partials/pdf-glossaries"; // Including .glossary
+@import "template/partials/pdf-highlighter";
+@import "template/partials/pdf-pullquotes"; // Supports .pullquote
+@import "template/partials/pdf-boxes";
+@import "template/partials/pdf-code"; // For computer code and preformatted text
+@import "template/partials/pdf-tables";
+@import "template/partials/pdf-figures";
+@import "template/partials/pdf-notes"; // Footnotes, endnotes, sidenotes // CRASHES PRINCE
```

Comment:
Do we need to add more detail (either here, or in pdf-notes.scss) about what might lead to this crash?

Reply:
Thanks, this was resolved, so I'll remove that debugging note. (If a single sidenote is longer than can fit on one page, Prince crashes. So it's caused by content, not CSS.)

Reply:
Done in [1f2fa57](https://github.com/electricbookworks/electric-book/pull/664/commits/1f2fa57d8a08924cbbae47e1f99205205af7c3a4).